Libraries/Dijkstra/Dijkstra.h

```diff
@@ -0,0 +1,262 @@
+#include "Map.h"
+
+struct coord {
+  uint8_t x, y;
+};
+
+struct dijkstra_tile {
+  uint8_t weight;
+  coord prev, current;
```

Comment:
don't you think it's redundant to have current?

Reply:
Uum, I'm not pretty sure why. I use current.